BedrockCommand.cpp

```diff
@@ -147,18 +149,24 @@ void BedrockCommand::finalizeTimingInfo() {
     for (const auto& entry: timingInfo) {
         if (get<0>(entry) == PREPEEK) {
             prePeekTotal += get<2>(entry) - get<1>(entry);
+        } else if (get<0>(entry) == BLOCKING_PREPEEK) {
+            prePeekTotal += get<2>(entry) - get<1>(entry);
+            blockingPrePeekTotal += get<2>(entry) - get<1>(entry);
```

Comment:
Do we want this (and for postProcess) to include both? I get why it's done, but I'm not sure it's clearer.

Reply:
Are you suggesting we should not do `prePeekTotal += get<2>(entry) - get<1>(entry);` and we should just add it up when we create the graph (in graphana) if we are interested in adding up?

Reply:
The idea would be to treat prePeek as `nonblocking prepeek`

Reply:
> The idea would be to treat prePeek as `nonblocking prepeek`

Does that mean that `prePeekTotal` should not contain blocking time?